x/ccv/provider/keeper/genesis_test.go

```diff
@@ -17,14 +16,14 @@ func (suite *KeeperTestSuite) TestGenesis() {
 
 	genState := suite.providerChain.App.(*appProvider.App).ProviderKeeper.ExportGenesis(suite.providerChain.GetContext())
 
-	suite.consumerChain.App.(*appConsumer.App).ProviderKeeper.InitGenesis(suite.consumerChain.GetContext(), genState)
+	suite.consumerChain.App.(*appProvider.App).ProviderKeeper.InitGenesis(suite.consumerChain.GetContext(), genState)
 
 	ctx = suite.consumerChain.GetContext()
 	for i := 0; i < 4; i++ {
 		expectedChainId := fmt.Sprintf("chainid-%d", i)
 		expectedChannelId := fmt.Sprintf("channelid-%d", i)
-		channelID, channelOk := suite.consumerChain.App.(*appConsumer.App).ProviderKeeper.GetChainToChannel(ctx, expectedChainId)
-		chainID, chainOk := suite.consumerChain.App.(*appConsumer.App).ProviderKeeper.GetChannelToChain(ctx, expectedChannelId)
+		channelID, channelOk := suite.consumerChain.App.(*appProvider.App).ProviderKeeper.GetChainToChannel(ctx, expectedChainId)
```

Comment:
this left me confused. why is there a need for a consumer to have the provider keeper?

Reply:
This test is definitely outdated.

Is it still required @AdityaSripal ?

Reply:
It should not be. Originally the tests were written with simapp having both consumer and provider hooked up to it. This may be a holdover

Reply:
Yeah the simapps are now seperate too so it should be possible to delete this.

Reply:
ah okay thank you